DetectorDescription/DDCMS/src/DDFilteredView.cc

```diff
@@ -552,6 +552,9 @@ const std::vector<double> DDFilteredView::parameters() const {
 
 const cms::DDSolidShape DDFilteredView::shape() const {
   assert(node_);
+  if ((volume().volume())->IsAssembly()) {
+    return (cms::DDSolidShape::ddbox); // Return dummy box
```

Comment:
@cvuosalo - it should return `cms::DDSolidShape::ddshapelesssolid`

Reply:
That would be better than a Box. This change will be made shortly.

Reply:
Actually, a Box is needed for consistency and to pass the DDSolid.cppunit.cc unit test.